bindings/bindingtester/spec/tenantTester.md

```diff
@@ -38,6 +38,12 @@ The tenant API introduces some new operations:
 
     Unsets the active tenant.
 
+#### TENANT_LIST
+
+    Pops the top 3 items off of the stack as BEGIN, END, & LIMIT. Returns list
+    of tenant names contained in the range BEGIN to END, numbering LIMIT at most.
+    May optionally push a future onto the stack.
```

Comment:
This should specify what is done with the result. I think currently you have it concatenating the results into one long byte string, though for consistency with other operations I would recommend creating a tuple and packing it.

See for example the spec for `GET_RANGE`.